discovery_engine/lib/src/generators/visitors/const_member_name_visitor.dart

```diff
@@ -0,0 +1,27 @@
+// Copyright 2022 Xayn AG
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, version 3.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import 'package:analyzer/dart/element/element.dart';
+import 'package:analyzer/dart/element/visitor.dart';
+
+class ConstMemberNameVisitor extends SimpleElementVisitor<void> {
+  var constMemberNames = List<String>.empty(growable: true);
```

Comment:
it is a fixed type, it's infered from `List<String>.empty()`. But we could change this to something like:

```dart
final constMemberNames = <String>[];
```
We can use `final` because we don't reassign the variable, we just mutate it.

Reply:
`final` is even better 👍